api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2587,7 +2588,7 @@ public void composeObject(
         size -= src.offset();
       }
 
-      if (size < PutObjectOptions.MIN_MULTIPART_SIZE
+      if (size < ObjectWriteArgs.MIN_MULTIPART_SIZE
```

Comment:
Some of the validations here are the same as those in `PutObjectArgs`. Might be a good idea to put them in a common method and call from both places.

Reply:
This will be taken care when we add arg buildet to `composeObject()`